packages/web/src/components/spa/JobsSection/jobs_section.css

```diff
@@ -1,5 +1,436 @@
+    text-align: justify;
+}
+
+.text span {
+    font-family: var(--font-family);
+    font-size: 1em;
+    color: var(--text-color);
+    font-weight: 1000;
+}
+
+.text-2 span {
+    font-family: var(--font-family);
+    font-size: 1em;
+    color: grey;
+}
+
+.button-container button {
+    border-radius: 4vh;
+}
+
+.button-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    justify-items: end;
+    padding-top: 5%;
+    padding-left: 5%;
+    padding-right: 5%;
+    align-items: center;
+}
+
+.button-container span {
+    height: 3.9vh;
+    width: 6vw;
+    font-size: 0.9em;
+    font-weight: 400;
+    color: black;
+    text-align: center;
+    margin-top: 3%;
+    padding-top: 1.5%;
+    box-shadow: 0 4vh 15vh 0 rgba(65, 132, 234, 0.75);
+    transition: all 0.3s ease-in-out;
+    border-radius: 6%;
+    background-image: rgb(215, 215, 217);
+}
+
+.button-container span:hover {
+    transform: translateY(-1vh);
+    background-color: #fff;
+    text-shadow: 1% 1% #4481eb;
+    border-radius: 10%;
+    box-shadow: 0 6vh 10vh 0 rgba(65, 132, 234, 0.95);
+    transition: all 0.3s ease-in-out;
+}
+
+button {
+    width: 20vw;
+    height: 6vh;
+    font-weight: 600;
+    color: #fff;
+    cursor: pointer;
+    text-align: center;
+    border: none;
+    background-size: 300% 100%;
+    font-size: 1em;
+    background-image: linear-gradient(
+        to right,
+        #32baf4,
+        #467edd,
+        #3f86ed,
+        #47acdf,
+        #24cc8e
+    );
+    box-shadow: 0 2% 1% 0 rgba(65, 132, 234, 0.75);
+    transition: all 0.3s ease-in-out;
+}
+
+button:hover {
+    background-position: right;
+    box-shadow: 0 6% 20% 0 rgba(65, 132, 234, 0.95);
+    transform: translateY(-2%);
+    background-color: #fff;
+    text-shadow: 1% 1% #4481eb;
+}
+
+@media only screen and (min-width: 1000px) and (max-width: 1389px) {
+    .jobs-container {
+        height: 118vh;
+        width: 101vw;
+    }
+
+    .job-box {
+        height: 50vh;
+        width: 26vw;
+    }
+
+    button {
+        width: 100vw;
+        height: 7vh;
+        font-size: 1.4em !important;
+    }
+
+    .button-container span {
+        height: 4.5vh;
+        width: 7vw;
+        font-size: 0.8em !important;
+        font-weight: 500;
+    }
+
+    .photo img {
+        width: 24vw;
+        height: 12vh;
+    }
+}
+
+@media only screen and (min-width: 1000px) and (max-width: 1035px) {
+    .jobs-container {
+        height: 100vh;
+        width: 101vw;
+    }
+
+    .job-box {
+        width: 26vw;
+        height: 40vh;
+    }
+
+    .text,
+    .text-2 {
+        font-size: 0.8em;
+        text-align: justify;
+    }
+
+    .button-container span {
+        font-size: 0.7em !important;
+        height: 3.5vh;
+    }
+
+    button {
+        font-size: 1.3em !important;
+    }
+}
+
+@media only screen and (min-width: 600px) and (max-width: 999px) {
+    .jobs-container {
+        height: 405vh;
```

Comment:
remove heights from queries and instead you should set a global `min-height` for the container